### PR TITLE
Fix performance test package naming for classpath

### PR DESCRIPTION
**Pull request by Blackbaud-JosephZelada**
@blackbaud/java-internal-tooling
Changing top package name from performanceTest to main so our gatling simulations show up on the classpath

### CODE REVIEW

**Comment by Blackbaud-MikeLueders** [comment]
tests need to be greenwashed

File: src/main/groovy/com/blackbaud/templates/project/BasicProject.groovy [hunk 1]
```diff
@@ -97,7 +94,7 @@ class BasicProject {
                            packageName: servicePackage,
                            className: testClassName
         }
-        applyTemplate("performance-test/src/performanceTest/scala/${servicePackagePath}") {
+        applyTemplate("performance-test/src/main/scala/${servicePackagePath}") {
```
Comment (Blackbaud-BethRichardson) [inline src/main/groovy/com/blackbaud/templates/project/BasicProject.groovy:97]: I thought we wanted it in a sub-module?

Comment (Blackbaud-BethRichardson) [reply src/main/groovy/com/blackbaud/templates/project/BasicProject.groovy:97]: realize this is under the submodule. doh

File: src/main/groovy/com/blackbaud/templates/project/PerformanceTestsProject.groovy [hunk 2]
```diff
@@ -15,6 +15,10 @@ class PerformanceTestsProject {
 
 # Performance test results
 results/
+""")
+        File gradleSettingsFile = basicProject.getProjectFileOrFail("settings.gradle")
```
Comment (Blackbaud-MikeLueders) [inline src/main/groovy/com/blackbaud/templates/project/PerformanceTestsProject.groovy:19]: settings.gradle is not guaranteed to exist

**Review by Blackbaud-MikeLueders — CHANGES_REQUESTED** [review]
run tests and ensure they pass (run `gw greenwashTemplateTests`)

File: src/main/groovy/com/blackbaud/templates/project/PerformanceTestsProject.groovy (hunk 2 above)
Comment (Blackbaud-JosephZelada) [reply src/main/groovy/com/blackbaud/templates/project/PerformanceTestsProject.groovy:19]: Fixed

**Comment by Blackbaud-JosephZelada** [comment]
greenwashTemplateTests passes for me locally

**Review by Blackbaud-MikeLueders — APPROVED** [review]

**Pull request merged**